src/topology/algebra/valuation.lean

```diff
@@ -13,19 +13,18 @@ import ring_theory.valuation.basic
 
 In this file, we define the non archimedean topology induced by a valuation on a ring.
 The main definition is a `valued` type class which equips a ring with a valuation taking
-values in a group with zero (living in the same universe). Other instances are then deduced from
-this.
+values in a group with zero. Other instances are then deduced from this.
 -/
 
 open_locale classical topological_space
 open set valuation
 noncomputable theory
 
-universe u
+universes u v
```

Comment:
It would be better to switch the order of `u` and `v` here, so that you can write `valued.{v} R` (the universe `u` would be inferred from that of `R`).

Reply:
Ok, I'll change it.